src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/SvcutilBootstrapper/SvcutilBootstrapper.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="dotnet-svcutil-lib">
-      <HintPath>$resultPath$/TestResults/TFMBootstrap/tfm100/bin/Debug/netcoreapp3.1/dotnet-svcutil-lib.dll</HintPath>
+      <HintPath>$resultPath$/TestResults/TFMBootstrap/tfm100/bin/Debug/net5.0/dotnet-svcutil-lib.dll</HintPath>
```

Comment:
With this change, will there be any issue for this to run on 3.1?  Will there be any backward compatibility issue?

Reply:
The baseline is fixed in a followed commit.